### ISSUE TITLE
re-frisk-rn display page without content

### ISSUE THREAD

**Issue by markokocic** [flexsurfer/re-frisk-rn#2]
I followed instructions to set up `re-frisk-rn` but in both cases I get only to the opening web page that doesn't provide any information about app-db, events or anything.

I'm using the latest react-native and shadow-cljs and android emulator.

Do you maybe have some minimal complete working example?

**Comment by flexsurfer** [comment]
make sure your app can connect to localhost:4567

**Comment by flexsurfer** [comment]
and also that you have preloads for re-frisk

**Comment by markokocic** [comment]
Proload is defined, when I access http://localhost:4567 I reach the page, but see only page skeleton, without any event, db or subscription data.

I don't see any exceptions in the application logs.

**Comment by flexsurfer** [comment]
please try to open http://localhost:4567 in your mobile device browser just to make sure device has access  to it

**Comment by markokocic** [comment]
good catch. Yes, I can access http://localhost:4567 on my laptop, but not from the device browser. For some reason device/emulator decides to use 192.168.0.111 as my host ip.

After executing `adp revers tcp;4567 tcp:4567` re-frisk started working, and I'm able to see events, subscriptions ... on my desktop browser.

However, the question is why I need to do port forwarding?

Application running on device somehow recognizes that it needs to connect to http://192.168.0.111:8081 to fetch compiled javascript code and shadow-cljs hot reloading also works without any port forwarding.

Now I have a workaround, but maybe one enhancement would be that re-frisk-rn uses the same IP as metro loader by default, so that port forwarding would not be required.

**Comment by flexsurfer** [comment]
thanks for your feedback, I'll take a look
